.github/workflows/yaspeller.yml

```
@@ -17,4 +17,14 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npx yaspeller {html,css,js,tools}/**/*.md
+      - name: Проверка правописания
+        run: npx yaspeller {html,css,js,tools}/**/*.md
+      - name: Сортировка словаря исключений
+        run: node .github/scripts/sort.js
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Отсортировать словарь исключений
```

```suggestion
          commit_message: Сортирует словарь исключений
```
Посмотрел историю и у нас как-то сложилось, что коммит говорит, что он делает.

---

у нас есть соглашение, что пишем коммиты в таком формате, а пул реквесте в формате «что делает?»

Я приму, но давай поднимем этот вопрос на созвоне или в чате

---

> Алёна писала соглашение

Я его как-то пропустил, давай обсудим, ага